Sources/GRPCHTTP2TransportNIOPosix/HTTP2ServerTransport+Posix.swift

```diff
@@ -294,7 +294,7 @@ extension HTTP2ServerTransport {
 
 @available(macOS 15.0, iOS 18.0, watchOS 11.0, tvOS 18.0, visionOS 2.0, *)
 extension HTTP2ServerTransport.Posix {
-  /// Configuration for the ``GRPCHTTP2TransportNIOPosix/GRPCHTTP2Core/HTTP2ServerTransport/Posix``.
+  /// Config for the `Posix` transport.
```

Comment:
I think having the links to the types makes navigation easier if you're going over it. If it's a matter of making it less "verbose", can we at least add a `- SeeAlso:` section (here and in the few other places we've removed links)?

Reply:
I agree but DocC gets trips over these and emits warnings. I think the problem stems from the namespace being defined in `GRPCHTTP2Core` and the type being defined in an extension in `GRPCHTTP2TransportNIOPosix`.

Reply:
Ah, I see that's sad :( 
Okay then, ignore this, nothing we can do.